easybuild/easyconfigs/b/Bison/Bison-2.7-ClangGCC-1.7.20.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'Bison'
+version = '2.7'
```

Comment:
I'd rather avoid version creep unless there is a known benefit, no? (yeah, we still luck a rule in this)